compiler/testData/codegen/box/properties/lateinit/local/implicitLocalLateinit.kt

```diff
@@ -0,0 +1,7 @@
+fun box(): String {
```

Comment:
This test is redundant: lateinit logic exists in localLateinit.kt, non lateinit in exactlyOnceCrossinline.kt and definiteValInitialization.kt

Reply:
This test is much simplier than exactlyOnceCrossinline.kt and definiteValInitialization.kt, though, both of which also check contracts and inline functions (and fail on JVM_IR right now due to other inliner bugs).